src/store/modules/order/actions.ts

```diff
@@ -0,0 +1,66 @@
+import { ActionTree } from 'vuex'
+import RootState from '@/store/RootState'
+import OrderState from './orderState'
+import * as types from './mutation-types'
+import { OrderService } from '@/services/OrderService'
+import { hasError, showToast } from '@/utils'
+import { translate } from '@/i18n'
+import { DateTime } from 'luxon'
+
+const actions: ActionTree<OrderState, RootState> = {
+  async getDraftOrder ({ commit, dispatch }, params) {
+    const payload = {
+      draftOrderId: params.id,
+      shopifyConfigId: params.shopifyConfigId
+    }
+    try {
+      const resp = await OrderService.getDraftOrder(payload);
+      if (resp.status === 200 && !hasError(resp) && resp.data.response.draft_order) {
```

Comment:
Add condition that whether response is present or not before accessing `draft_order`.